src/browser/external_window_event_adapter.js

```diff
@@ -37,120 +38,127 @@ class ExternalWindowEventAdapter {
 
         let cachedState = null;
 
-        ofEvents.on(route.externalWindow('focus', uuid, name), () => {
-            browserWindow.emit('focus');
-        });
-
-        ofEvents.on(route.externalWindow('blur', uuid, name), () => {
-            browserWindow.emit('blur');
-        });
-
-        ofEvents.on(route.externalWindow('state-change', uuid, name), () => {
-            let prevState = cachedState || 'normal';
-
-            let currState = 'normal';
-            if (browserWindow.isMinimized()) {
-                currState = 'minimized';
-            } else if (browserWindow.isMaximized()) {
-                currState = 'maximized';
-            }
-
-            if (prevState !== currState) {
-                if (currState === 'minimized') {
-                    browserWindow.emit('minimize');
-                } else if (currState === 'maximized') {
-                    browserWindow.emit('maximize');
-                } else {
-                    browserWindow.emit('restore');
+        const eventMap = {
+            'focus': () => {
+                if (!browserWindow.isDestroyed()) {
+                    browserWindow.emit('focus');
```

Comment:
I think this has been fixed - external events were not being cleaned up.  The only changes to this file are unsubscribe